src/content/article/kodi-21-0-omega-beta-3/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Kodi 21.0 "Omega" Beta 3
+date: 2024-02-16T11:21:58.850Z
+author: Team Kodi
+tags:
+  - Developer
+  - Prerelease
+featured_image:
+  src: /images/blog/kodi_splash_v21.0_omega_beta_1080p.webp
+  title: Kodi 21 "Omega" Splash Screen
+  alt: Kodi 21 "Omega" Splash Screen - the Kodi logo sits at the centre of a black
+    screen, a stylised Greek letter "Omega" around it.
+---
+Time marches on for Kodi 21 "Omega", we are here with the third Beta release!
+
+This is a major release, so, as you'd expect, it comes with many new features. However, with a core change to FFmpeg 6.0, there is a very good chance that some regressions may occur.  As such, please be aware that installing this will most likely break things, and we'd really like your help at that point to identify the problems and get them fixed. Please raise [Github issues](https://github.com/xbmc/xbmc/issues) with full debug logs to help us resolve any issues you wonderful testers come across.
```

Comment:
```suggestion
This is a major release, so, as you'd expect, it comes with many new features. However, with a core change to FFmpeg 6.0, there is a very good chance that some regressions may occur.  As such, please be aware that installing this will most likely break things, and we'd really like your help to identify any problems and get them fixed. Please raise [GitHub issues](https://github.com/xbmc/xbmc/issues) with full debug logs to help us resolve any issues you wonderful testers come across.
```